addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -217,7 +217,7 @@ website.if_dom_contains('.oe_slide_js_upload', function () {
                         tags = {results: []};
                     _.each(data, function (obj) {
                         if (that.matcher(query.term, obj.name)) {
-                            tags.results.push({id: obj.id, text: obj.name });
+                            tags.results.push({id: "_" + $.trim(obj.name.toLowerCase()), text: obj.name });
```

Comment:
do not pass tag name instead of id, each tag has unique and it will attache to slide using  [4, tag_id] on create, this change will raise error on create of slide